### PR TITLE
zh-translation/docs/reference/config/proxy_extensions/wasm_telemetry/index.md

### PR DESCRIPTION
**Pull request by k-9527**
[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/reference/config/proxy_extensions/wasm_telemetry/index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Wasm-based Telemetry [Experimental]
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/proxy_extensions/wasm_telemetry/index.md:2]: 基于 Wasm 的遥测[实验特性]

File: content/zh/docs/reference/config/proxy_extensions/wasm_telemetry/index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+description: 如何在 Wasm 运行时环境下启用遥测生成(实验性质)。
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/proxy_extensions/wasm_telemetry/index.md:3]: 如何使用 Wasm 进行遥测。

File: content/zh/docs/reference/config/proxy_extensions/wasm_telemetry/index.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    - /zh/docs/reference/config/telemetry/telemetry_v2_with_wasm/
+---
+
+默认情况下，遥测生成特性作为 Istio 代理过滤器启用。同样的过滤器也被编译成 WebAssembly (Wasm) 模块，并随 Istio 代理一起发布。
+
+{{< text bash >}}
+$ istioctl install --set profile=preview
+{{< /text >}}
+
+或者，设置以下两个值，使用 `default` 配置文件启用基于 Wasm 的遥测技术:
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/proxy_extensions/wasm_telemetry/index.md:17]: 设置以下两个参数。

File: content/zh/docs/reference/config/proxy_extensions/wasm_telemetry/index.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+title: Wasm-based Telemetry [Experimental]
+description: 如何在 Wasm 运行时环境下启用遥测生成(实验性质)。
+weight: 60
+owner: istio/wg-policies-and-telemetry-maintainers
+test: no
+aliases:
+    - /zh/docs/reference/config/telemetry/telemetry_v2_with_wasm/
+---
+
+默认情况下，遥测生成特性作为 Istio 代理过滤器启用。同样的过滤器也被编译成 WebAssembly (Wasm) 模块，并随 Istio 代理一起发布。
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/proxy_extensions/wasm_telemetry/index.md:11]: 过滤器可以不翻译。

File: content/zh/docs/reference/config/proxy_extensions/wasm_telemetry/index.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+title: 基于 Wasm 的遥测[实验特性]
+description: 如何使用 Wasm 进行遥测。
+weight: 60
+owner: istio/wg-policies-and-telemetry-maintainers
+test: no
+aliases:
+    - /zh/docs/reference/config/telemetry/telemetry_v2_with_wasm/
+---
+
+默认情况下，遥测生成特性作为 Istio 代理启用。同时也被编译成 WebAssembly (Wasm) 模块，并随 Istio 代理一起发布。
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/proxy_extensions/wasm_telemetry/index.md:11]:
```suggestion
默认情况下，遥测默认启用并作为一个 `filter` 被编译在 Istio 代理中，同时也被编译成 WebAssembly (Wasm) 模块，并随 Istio proxy 一起发布。
```

Comment (kebe7jun) [reply content/zh/docs/reference/config/proxy_extensions/wasm_telemetry/index.md:11]: 好像后面

File: content/zh/docs/reference/config/proxy_extensions/wasm_telemetry/index.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+$ istioctl install --set values.telemetry.v2.metadataExchange.wasmEnabled=true --set values.telemetry.v2.prometheus.wasmEnabled=true
+{{< /text >}}
+
+{{< warning >}}
+基于 Wasm 的遥测技术存在几个已知的局限性:
+
+* 代理 CPU 的使用将在 Wasm 模块加载阶段(例如：当上述配置被应用时)达到高峰。增加代理 CPU 资源限制将有助于加速加载。
+* 当代理基线资源使用增加时，根据初步的性能测试结果运行基于 Wasm 的遥测要比默认安装环境多花费30%~50%的 CPU，并使内存使用量增加一倍。
+
+性能将在接下来的版本中不断改进。
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/proxy_extensions/wasm_telemetry/index.md:29]:
```suggestion
性能问题将在接下来的版本中不断改进。
```

File: content/zh/docs/reference/config/proxy_extensions/wasm_telemetry/index.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    - /zh/docs/reference/config/telemetry/telemetry_v2_with_wasm/
+---
+
+默认情况下，遥测生成特性作为 Istio 代理启用。同时也被编译成 WebAssembly (Wasm) 模块，并随 Istio 代理一起发布。
+
+{{< text bash >}}
+$ istioctl install --set profile=preview
+{{< /text >}}
+
+或者，设置以下两个参数，使用 `default` 配置文件启用基于 Wasm 的遥测技术:
```
Comment (orangegzx) [inline content/zh/docs/reference/config/proxy_extensions/wasm_telemetry/index.md:17]:
```suggestion
或者，设置以下两个参数，使用 `default` 配置文件启用基于 Wasm 的遥测技术：
```

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**